### ISSUE TITLE
Render ready markdown text

### ISSUE THREAD

**Issue by lyf2000** [nasa8x/v-markdown-editor#11]
I don't actually understand how to initially render markdown text with markdown style, what do I suppose to add in `markdown-editor` template?
```
<markdown-editor
          toolbar=""
          v-model="content"
            ></markdown-editor>
```

**Comment by nasa8x** [comment]
You can read config: [https://morioh.com/p/2a1c9c34024b](https://morioh.com/p/2a1c9c34024b)

**Comment by BogdanFalk** [comment]
Indeed you can't have just preview if you do this:
`<markdown-editor
          toolbar=""
          v-model="content"
            ></markdown-editor>`

it will show just a box with the markdown normally without toolbar.

It should be a way to get to the preview variable inside, like adding something like preview-mode which would set that to true:

`<markdown-editor
          toolbar=""
          v-model="content"
          preview-mode="true"
            ></markdown-editor>`

Maybe there is one to access the **command** function but I can't figure it out
Cheers.

**Comment by SuryaVanLierde** [comment]
I agree, it would be very helpful if we could just pass an option to trigger the preview prop to be set to true on instantiation. I want two instances next to each other with the same model, so the second one shows a preview of what the user enters in the first one
